plugin/wakatime.vim

```diff
@@ -42,7 +42,7 @@ let s:VERSION = '8.0.1'
     if s:home == '$WAKATIME_HOME'
         let s:home = expand("$HOME")
     endif
-    let s:cli_location = substitute(substitute(expand("<sfile>:p:h"), '\', '/', 'g'), '/plugin$', '', '') . '/packages/wakatime/cli.py'
+    let s:cli_location = substitute(expand("<sfile>:p:h:h"), '\', '/', 'g') . '/packages/wakatime/cli.py'
```

Comment:
What's the extra `:h` for?

Reply:
Instead of calling substitute('/plugin$')  again, just to move up one dir, ```:h``` does that automatically.